### PR TITLE
[NET-11222] rollback command

### PR DESCRIPTION
**Pull request by jm96441n**
### Changes proposed in this PR ###  
- rollback the `sync-catalog purgeK8sServicesFromNode` command so we can implement the automatic removal on helm upgrade/install
-

### Checklist ###
- [ ] Tests added
- [X] [CHANGELOG entry added](https://github.com/hashicorp/consul-k8s/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

File: control-plane/subcommand/sync-catalog/command.go [hunk 1]
```diff
@@ -156,8 +156,8 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
-	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
-		"Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
+	// c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
+	// "Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
```
Comment (missylbytes) [inline control-plane/subcommand/sync-catalog/command.go:160]: We should remove instead of just comment out, right?

Comment (jm96441n) [reply control-plane/subcommand/sync-catalog/command.go]: yeah, we're going to be putting it back with some different behavior for  the next patch but it's probably better to just remove it now

File: control-plane/subcommand/sync-catalog/command.go [hunk 2]
```diff
@@ -156,8 +156,8 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
-	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
-		"Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
+	// c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
```
Comment (wangxinyi7) [inline control-plane/subcommand/sync-catalog/command.go]: Do we want to completely remove the code or only comment out the code?
We have unused flag in the above and unused function in the bottom

File: control-plane/subcommand/sync-catalog/command.go [hunk 3]
```diff
@@ -156,8 +156,6 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
-	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
-		"Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
 	c.flags.StringVar(&c.flagFilter, "filter", "",
```
Comment (wangxinyi7) [inline control-plane/subcommand/sync-catalog/command.go:159]: We also need to remove this flag if we want to rollback

**Review by sarahalsmiller — APPROVED** [review]

**Review by sophie-gairo — APPROVED** [review]

**Pull request merged**